wger/core/templates/template_features.html

```diff
@@ -25,6 +25,7 @@
     <meta charset="utf-8">
     <meta name="author" content="Roland Geider">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
```

Comment:
Font-awesome is already included by bower_components. Is it necessary to add it again using the CDN link?